forms/GroupedDropdownField.php

```diff
@@ -84,6 +87,9 @@ public function Field($properties = array()) {
 		return FormField::create_tag('select', $this->getAttributes(), $options);
```

Comment:
This may return a string, but it shouldn't so this needs to be wrapped in:

`DBField::create_field('HTMLText', ...)`